specification/appcomplianceautomation/resource-manager/Microsoft.AppComplianceAutomation/preview/2022-11-16-preview/appcomplianceautomation.json

```diff
@@ -0,0 +1,1374 @@
+              "x-ms-secret": true
+            }
+          }
+        },
+        "complianceDetailedPdfReport": {
+          "description": "compliance detailed pdf report",
+          "type": "object",
+          "readOnly": true,
+          "properties": {
+            "sasUri": {
+              "description": "uri of compliance detailed pdf report",
+              "type": "string",
+              "readOnly": true
+            }
+          }
+        }
+      }
+    },
+    "ComplianceReportItem": {
+      "type": "object",
+      "description": "Object that includes all the content for single compliance result.",
+      "properties": {
+        "categoryName": {
+          "type": "string",
+          "description": "The category name.",
+          "readOnly": true
+        },
+        "controlId": {
+          "type": "string",
+          "description": "The control Id.",
+          "readOnly": true
+        },
+        "controlName": {
+          "type": "string",
+          "description": "The control name.",
+          "readOnly": true
+        },
+        "controlType": {
+          "type": "string",
+          "description": "The control type.",
+          "$ref": "#/definitions/ControlType",
+          "readOnly": true
+        },
+        "complianceState": {
+          "type": "string",
+          "description": "The compliance result's status.",
+          "readOnly": true
+        },
+        "policyId": {
+          "type": "string",
+          "description": "The compliance result mapped policy Id.",
+          "readOnly": true
+        },
+        "policyDisplayName": {
+          "type": "string",
+          "description": "The policy's display name.",
+          "readOnly": true
+        },
+        "policyDescription": {
+          "type": "string",
+          "description": "The policy's detail description.",
+          "readOnly": true
+        },
+        "subscriptionId": {
+          "type": "string",
+          "description": "The compliance result mapped subscription Id.",
+          "readOnly": true
+        },
+        "resourceGroup": {
+          "type": "string",
+          "description": "The compliance result mapped resource group.",
+          "readOnly": true
+        },
+        "resourceType": {
+          "type": "string",
+          "description": "The compliance result mapped resource type.",
+          "readOnly": true
+        },
+        "resourceId": {
+          "type": "string",
+          "description": "The compliance result mapped resource Id.",
+          "readOnly": true
+        },
+        "statusChangeDate": {
+          "description": "The compliance result last changed date. For unavailable date, set it as N/A.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ProvisioningState": {
+      "type": "string",
+      "description": "Resource provisioning states.",
+      "enum": [
+        "Succeeded",
+        "Failed",
+        "Cancelled",
```

Comment:
>Cancelled [](http://example.com/codeflow?start=9&length=9)

I hear that provisioning state has to be spelled 'Canceled' with only one letter 'l' (L) to be properly recognized by ARM.

Reply:
@TimLovellSmith  Do you have the reference link for this recommendation?

Reply:
Fixed, "Cancelled" has been changed to "Canceled".